app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name="mobile.programming.musicapp.MainActivity">
+        <activity android:name="mobile.programming.musicApp.MainActivity">
```

Comment:
Here, I have also changed the multi-name package name to camelcase.

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase